app/constants/trezor.js

```diff
@@ -1,10 +1,12 @@
-export const model1_decred_homescreen =
+export const PIN_LABELS = "ABCDEFGHI";
```

Comment:
Could you switch this to "•••••••••" or otherwise get rid of PIN_LABELS altogether? Would allow closing #1932 and #2562 as well.